ray_julia_jll/gen/artifacts.jl

```diff
@@ -0,0 +1,137 @@
+using Base: SHA1, BinaryPlatforms
+using CodecZlib: GzipCompressorStream, GzipDecompressorStream
+using CURL_jll
+using jq_jll
+using LibGit2: LibGit2
+using Pkg.Artifacts: bind_artifact!
+using Pkg.Types: read_project
+using SHA: sha256
+using Tar
+using wget_jll
+
+const DIR = mktempdir()
+const GITHUB_URL = "https://api.github.com/repos"
+
+const TRIPLET_REGEX = r"""
+    ^ray_julia.v(?<jll_version>([0-9]\.){3})
+    (?<triplet>[a-z, 0-9, \-, \_,]+)
```

Comment:
This is equivalent. Do you want to match on commas? I left that out
```suggestion
    (?<triplet>[a-z0-9_-]+)
```